### DISCUSSION TITLE
About usage of API

### DISCUSSION THREAD

**Discussion by nmpasdanis** [open-meteo/open-meteo#141] [General]
Hello there, I develop a simple Weather Forecast application using your's Open Meter Api by C# and I would like to know if your url must have only double parameters e.g (2.31) not ( 2,31).
Thank you very much!!!

**Comment by patrick-zippenfenig** [comment]
Hi,

you have to use a dot for decimal numbers. A coma is considered a delimiter for a list: E.g. the elevation API uses comas for delimters: https://api.open-meteo.com/v1/elevation?latitude=52.52,48.85&longitude=13.41,2.35

Best, Patrick

**Comment by nmpasdanis** [comment]
Thank you Patrik!!
